modules/images-other-jenkins-agent-images.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins-agent.adoc
+
+[id="images-other-jenkins-agent-images_{context}"]
+= Jenkins agent images
+
+The {product-title} Jenkins agent images come in two varieties, Red Hat
+Enterprise Linux (RHEL) 7 based images and CentOS 7 Based Images.
+
+[discrete]
+=== RHEL 7 Based Images
+
+RHEL 7 Jenkins images are available through the Red Hat Registry:
+
+----
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
+----
+
+[NOTE]
+====
+The RHEL 7 Jenkins images exist as an imagestream in {product-title}.
+====
+
+[discrete]
+=== CentOS 7 Based Images
+
+These images are available on Docker Hub:
+
+----
+$ docker pull openshift/jenkins-agent-maven-35-centos7
+$ docker pull openshift/jenkins-agent-nodejs-8-centos7
```

Comment:
yeah new image names for 4.x ...see https://github.com/openshift/jenkins#installation-openshift-v4

providing both quay.io and registry.redhat.io versions would be good, similar to the centos/rhel distinction in 3.x

and someone should double check the names in the readme ... there was some churn during the 4.1 dev cycle